tox.ini

```diff
@@ -54,10 +54,8 @@ commands = make linkcheck
 
 [testenv:lint]
 deps =
-    flake8
-    isort
+    -rrequirements_dev.txt
     check_manifest
-skip_install = true
```

Comment:
No longer skipping install in testenv:lint?

Reply:
I expected `isort` not to recognized `factory` as a first party without it, but forgot to remove `known_first_party` from `config.cfg`. I removed it and restored `skip_install = true`, builds keep passing. :tada: